tests/lib/fakeinstaller/mk-classic-rootfs.sh

```diff
@@ -77,16 +79,23 @@ if [ ! -d "$DST" ]; then
     exit 1
 fi
 
-# extract the base
+# This script is either used as part of an installer image which will have
+# a "base.squashfs". Here very little additional setup is needed or as part
+# of a spread test in which case the installer needs to prepare the system
+# to be used from spread. The "ROLE" var will be set accordingly so that
+# the "prepare_classic_rootfs" knows what to do.
+ROLE="unset"
```

Comment:
nitpick: I would keep this like
```suggestion
ROLE=""
```